src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -153,16 +147,13 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
                 css={[BylineCss.link, BylineCss.authorLink]}
                 href={authorTopicUrl}
               >
-                <span
+                <Text
                   className="byline__link-text"
-                  css={[
-                    BylineCss.author,
-                    getSansBold(service),
-                    getBodyCopy(script),
-                  ]}
+                  fontVariant="sansBold"
+                  css={[BylineCss.author, getBodyCopy(script)]}
```

Comment:
In each of these new `<Text />` component uses, we can remove the `getBodyCopy, getBrevier` etc functions and make use of the `size=` prop on the Text component. That will allow us to get rid of the psammead package imports for getting those font size styles.